configs/prompt_muons_example.yaml

```diff
@@ -0,0 +1,58 @@
+chain_name: corsika_snowstorm_prompt_v1_5_1
+n_events_per_run: 100
+n_runs: 10000
+
+dataset_number: 30012
+seed: 3
+
+corsika_module: 'icecube.simprod.modules.Corsika5ComponentGenerator'
+corsika_settings: {
+
+    # things that are pirobably fine like this most of the time
+    'corsikaVersion': '77420',
+    'model': 'SIBYLL',
+    'CutoffType': 'EnergyPerParticle',
+    'oversampling': 1,
+    'trimshower': True,
+    'pnorm': [10., 5., 3., 2., 1.],
+    'pgam': [1., 1., 1., 1., 1.],
+    'UsePipe': True,
+    
+    'ecuts1': 273.,  # hadron min energy (see corsika docs)
+    'ecuts2': 273.,  # muon min energy (see corsika docs)
+    'ecuts3': 273.,  # electron min energy (see corsika docs)
+    'ecuts4': 273.,  # photon min energy (see corsika docs)
+    'CVMFS': '/data/user/pgutjahr/software/CORSIKA/',
```

Comment:
Needs to be moved to CVMFS

Reply:
Now it is available at `/net/cvmfs_users/pgutjahr/software/CORSIKA/`. Also a README how to build CORSIKA is added.

Reply:
@The-Ludwig, can you adjust the `CVMFS` path? Thanks